Delicate-Butterflies/Bangazon-Workforce
# Sb add employee

Related Issue: #2 

Description: Allows users to add a new employee to the database.

Chai/Mocha Tests File:

PR Testing Steps:
- sequelize db:migrate:undo:all
- pull down the branch
- sequelize db:migrate
- sequelize db:seed:all
- nodemon app.js
- go to /employees route
- click Add Employee button
- Enter information and submit
- You will be redirected to /employees - new information should appear here as well now

Check list before you make a PR:
